lib/appropriate_bodies/errors.rb

```diff
@@ -1,5 +1,7 @@
 module AppropriateBodies
   module Errors
-    class TeacherAlreadyClaimedError < StandardError; end
+    # class TeacherAlreadyClaimedError < StandardError; end
+    class TeacherHasActiveInductionPeriodWithABError < StandardError; end
```

Comment:
I think `Current` vs `Another` might make it a little clearer what the difference is.

```suggestion
    class TeacherHasActiveInductionPeriodWithCurrentABError < StandardError; end
```